lwgenerate/gcp/gcp.go

```diff
@@ -0,0 +1,505 @@
+package gcp
+
+import (
+	"github.com/hashicorp/hcl/v2/hclwrite"
+	"github.com/lacework/go-sdk/lwgenerate"
+	"github.com/pkg/errors"
+)
+
+type ExistingServiceAccountDetails struct {
+	// Existing Service Account Name
+	Name string
+
+	// Existing Service Account private key in JSON format, base64 encoded
+	Base64EncodedPrivateKey string
```

Comment:
Naming is a little too verbose? Given the comment about explains it's base64. Perhaps just PrivateKey?

Reply:
done